base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -521,21 +521,19 @@ where
 
     fn handle_base_node_service_event(&mut self, event: Arc<BaseNodeEvent>) {
         match (*event).clone() {
-            BaseNodeEvent::BaseNodeStateChanged(state) => {
-                let trigger_validation = match (self.last_seen_tip_height, state.chain_metadata.clone()) {
-                    (Some(last_seen_tip_height), Some(cm)) => last_seen_tip_height != cm.height_of_longest_chain(),
-                    (None, _) => true,
-                    _ => false,
-                };
-                if trigger_validation {
-                    let _id = self.validate_outputs().map_err(|e| {
-                        warn!(target: LOG_TARGET, "Error validating  txos: {:?}", e);
-                        e
-                    });
-                }
-                self.last_seen_tip_height = state.chain_metadata.map(|cm| cm.height_of_longest_chain());
+            BaseNodeEvent::BaseNodeStateChanged(_state) => {
```

Comment:
This will stop the wallet from refreshing on node sync done, which might not be ideal

Reply:
This state is published when only the latency changes, so it was triggering much more frequently than needed

Reply:
See related comment ^^. I think we should add a `ConnectedBaseNodeChanged` event so we can refresh on that specifically.

Reply:
Resolved with comment below